Improve API Docs (#98)
ballerina-platform/module-ballerina-regex: regex-ballerina/Package.md

## Package Overview

> The Regex library is one of the standard libraries of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.

Author: ```suggestion
The `regex` library is one of the standard libraries of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```

> This Regex library is one of the standard libraries of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.

Reviewer: ```suggestion
The Regex library is one of the standard libraries of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
```

> This package provides functions for regex functionalities like checking whether a string matches a given regex, replacing substrings, and splitting strings based on a regex.

Author: ```suggestion
This package provides functions for regex functionalities such as checking whether a string matches a given regex, replacing substrings, and splitting strings based on a regex.
```
